.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/3rdParty/babylon"]
+	path = test/3rdParty/babylon
+	url = https://github.com/babel/babylon.git
```

Comment:
@hzoo we need the text fixtures, and they're not shipped with babylon. Also I notice you've submoduled